packages/reactive-element/src/test/decorators-modern/README.md

```diff
@@ -0,0 +1,6 @@
+This is a set of tests that use "modern" decorator syntax (`accessor`, etc) that are run with both experimental and standard decorators to ensure identical behavior between the experimental and standard decorator code paths.
```

Comment:
Technically the behavior is not identical because there is a slight difference in initial `undefined` values? But maybe not worth documenting here?